src/Numerics/project.json

```diff
@@ -0,0 +1,97 @@
+﻿{
+  "name": "MathNet.Numerics.Core",
+  "title": "Math.NET Numerics",
+  "description": "Math.NET Numerics is the numerical foundation of the Math.NET project, aiming to provide methods and algorithms for numerical computations in science, engineering and every day use.",
+  "authors": [ "Christoph Ruegg", "Marcus Cuda", "Jurgen Van Gael" ],
+
+  "version": "3.17.0",
+  "dependencies": {
+
+  },
+
+  "frameworks": {
+    "netstandard1.0": {
+      "imports": "dnxcore50",
+      "dependencies": {
+        "System.Collections": "4.3.0",
+        "System.Diagnostics.Debug": "4.3.0",
+        "System.Diagnostics.Tools": "4.3.0",
+        "System.Globalization": "4.3.0",
+        "System.IO.FileSystem": "4.3.0",
+        "System.Linq": "4.3.0",
+        "System.Reflection": "4.3.0",
+        "System.Resources.ResourceManager": "4.3.0",
+        "System.Runtime": "4.3.0",
+        "System.Runtime.Extensions": "4.3.0",
+        "System.Runtime.InteropServices": "4.3.0",
+        "System.Runtime.InteropServices.RuntimeInformation": "4.3.0",
+        "System.Runtime.Numerics": "4.3.0",
+        "System.Runtime.Serialization.Primitives": "4.3.0",
+        "System.Security.Cryptography.Algorithms": "4.3.0",
+        "System.Threading": "4.3.0",
+        "System.Threading.Tasks.Parallel": "4.3.0"
+      },
+
+      "buildOptions": {
+        "define": [ "NETSTANDARD1_0", "NOSERIALIZATION", "NET45REFLECTION", "NOCLONABLE" ],
+        "optimize": true
+      }
+    },
+
+    "netstandard1.3": {
+      "imports": "dnxcore50",
+      "dependencies": {
+        "System.AppContext": "4.3.0",
+        "System.Collections": "4.3.0",
+        "System.Diagnostics.Debug": "4.3.0",
+        "System.Diagnostics.Tools": "4.3.0",
+        "System.Globalization": "4.3.0",
+        "System.IO.FileSystem": "4.3.0",
+        "System.Linq": "4.3.0",
+        "System.Reflection": "4.3.0",
+        "System.Resources.ResourceManager": "4.3.0",
+        "System.Runtime": "4.3.0",
+        "System.Runtime.Extensions": "4.3.0",
+        "System.Runtime.InteropServices": "4.3.0",
+        "System.Runtime.InteropServices.RuntimeInformation": "4.3.0",
+        "System.Runtime.Numerics": "4.3.0",
+        "System.Runtime.Serialization.Primitives": "4.3.0",
+        "System.Security.Cryptography.Algorithms": "4.3.0",
+        "System.Threading": "4.3.0",
+        "System.Threading.Tasks.Parallel": "4.3.0",
+        "Microsoft.Win32.Primitives": "4.3.0"
+      },
+
+      "buildOptions": {
+        "define": [ "NETSTANDARD1_3", "NOSERIALIZATION", "NOCLONABLE", "NET45REFLECTION", "NATIVE" ],
+        "optimize": true
+      }
+    },
+
+    "net35": {
+      "dependencies": {
+        "TaskParallelLibrary": "1.0.2856"
+      },
+      "frameworkAssemblies": {
+        "System.Runtime.Serialization": "3.0.0.0"
+      },
+
+      "buildOptions": {
+        "define": [ "NET35", "NOSYSNUMERICS" ],
+        "optimize": true
+      }
+    },
+
+    "net451": {
```

Comment:
We currently do not yet explicitly build for .Net 4.5.1, but we have a build for .Net 4.0 and 4 PCL builds. Is this supported by this approach as well? (No matter if not, not worth investing time, just wondering)